seqr/views/utils/dataset_utils.py

```diff
@@ -304,6 +304,17 @@ def _parse_tsv_row(row):
 
 RNA_OUTLIER_COLUMNS = {'geneID': 'gene_id', 'pValue': 'p_value', 'padjust': 'p_adjust', 'zScore': 'z_score'}
 
+RNA_SPLICE_OUTLIER_COLUMNS = {
+                                 'geneID': 'gene_id',
+                                 'pValue': 'p_value',
+                                 'zScore': 'z_score',
+                                 'chrom': 'chrom',
+                                 'start': 'start',
+                                 'end': 'end',
+                                 'strand': 'strand',
+                                 'reads': 'read_count'
```

Comment:
not super important, but if you change the model field to `read_count` instead of `reads` you could do `_to_camel_case` for the mapped key instead of needing to set it as an explicit mapping, and also it would be a more accurate field name